server_test.go

```diff
@@ -1506,6 +1506,37 @@ func TestServer_LimitAndOffset(t *testing.T) {
 	}
 }
 
+// Ensure the server can execute a group by with a fill(0) and very large limit
+// that results in a grouping of over 100,000 points returns an error
+func TestServer_ExecuteGroupByFillLimit(t *testing.T) {
```

Comment:
I was headed that direction and realized that those tests aren't set up to check for error conditions.  I do agree it should be there.  I'll have to do some refactoring of that test suite to make this one possible.